content/en/hosting/kubernetes-vs-docker.md

```diff
@@ -2,48 +2,118 @@
+
+k8s offers:
+
+* Resilient network across either physical or VM nodes in the cluster. This allows strong distribution of the heavy CPU and RAM loads that CouchDB can incur under heavy use.
 * Orchestration of multiple deployments ensuring to allow easy hosting of multi-tenants. For example you may opt to have a user acceptance testing (UAT), staging and production instances all in one cluster.
 * Service discovery which  enables to route public requests to deployments within the cluster
 * Integration with hypervisors such as VMWare and ESX. This is possible due to CRD support in Kubernetes that allows 3rd parties to create integrations
 * Helm support and integration that makes it easy to easily deploy applications on to the cluster
 * Highly efficient snapshot backups when used with a storage area network [SAN](https://en.wikipedia.org/wiki/Storage_area_network) or Amazon's [Elastic Block Storage](https://aws.amazon.com/ebs/) (EBS).
 
-The main components of a Kubernetes deployment include:
+The main components of a Kubernetes CHT deployment include one less service than a Compose based deployment as ingress via `nginx` isn't needed as it's handled externally:
 
 * A pod for each CouchDB instance.
 * A CHT API pod.
 * A CHT HAProxy Healthcheck pod.
 * A CHT HAProxy pod.
 * Upgrade Service pod.
 * CHT-Sentinel pod.
-## Docker Compose
 
-The Docker Compose based CHT Core deployment  was Medic's first attempt to make CHT 4.x cloud native.  The Compose files work quite well for application developer setups on laptops and the like (check out the [Docker Helper]({{< relref "hosting/4.x/app-developer#cht-docker-helper-for-4x" >}})!). Additionally, we have existing published guides on how to deploy single and multi-node Compose based solutions.  For small, low use instances, likely the single node Compose deployments will be fine.  We do not recommend setting up a multi-node deployment on Compose with an overlay network. Use Kubernetes instead for a more stable and [horizontally scalable]({{< relref "hosting/vertical-vs-horizontal" >}}) solution.
+## Example deployments
 
-Like Kubernetes above, Docker Compose deploys the same services but adds an additional 7th to allow for ingress/egress via a reverse proxy:
+Below are examples from 2 real world Ministries of Health (MoH) which have been anonymized and Medic hosted instances.  Users are total provisioned users, fewer users may be active in a given month.
+
+### MoH Example 1
+
+This MoH hosts 47 production instances in a national data center running [K3s](https://k3s.io/). They run a simplified k3s set up with 47 isolated installations of k3s to easy management complexity.  Thus each CHT instance is bound to the VM's local storage.  If a VM fails, there is no automatic fail-over and a restore from backup is needed.
+
+|                   |                             |
+|------------------:|:----------------------------|
+|    CHT Instances: | 47                          |
```

Comment:
I think the more important breakdown here was by instance, and how many users / resources / repication apdex they have, at least that's what I was envisioning.

Reply:
Do you mean for all three examples, or for just Medic hosted examples?  Assuming just medic hosted, maybe breaking out two examples deployments, one small and one big?